perllib/FixMyStreet/App/Controller/Waste.pm

```diff
@@ -679,6 +686,7 @@ sub calendar : Chained('property') : Args(0) {
 sub calendar_ics : Chained('property') : PathPart('calendar.ics') : Args(0) {
     my ($self, $c) = @_;
     $c->res->header(Content_Type => 'text/calendar');
+    $c->res->header(Cache_Control => 'max-age=86400');
```

Comment:
If we wanted easier control over the timing, this could be in the server's httpd.conf, but is also fine here.

Reply:
Thought it made sense here because we also set the TTL within the iCal file a few lines below so keeps it all together.